source/jsonnet/england-wales/household/blocks/who-lives-here/any_more_people_living_here.jsonnet

```diff
@@ -70,19 +70,19 @@ local editQuestion(questionTitle) = {
 
 
 {
-  id: 'anyone-else-temp-away-list-collector',
+  id: 'any-more-people-living-here',
   type: 'ListCollector',
   for_list: 'household',
   add_answer: {
-    id: 'anyone-else-temp-away-answer',
+    id: 'any-more-people-living-here-answer',
     value: 'Yes, I need to add someone',
   },
   remove_answer: {
-    id: 'anyone-else-temp-away-remove-confirmation',
+    id: 'any-more-people-living-here-remove-person-confirmation',
     value: 'Yes, I want to remove this person',
   },
   question: {
-    id: 'anyone-else-temp-away-confirmation-question',
+    id: 'any-more-people-living-here-confirmation-question',
```

Comment:
It seems like the question here should be `any-more-people-living-here-question` to maintain the consistency between other question changes. Doesn't matter on the type.

Reply:
I changed these following @petechd comment above, so that it is in-line with the changes made for the household id's in the CCS PR, as there was some discussion about this there [PR](https://github.com/ONSdigital/eq-questionnaire-schemas/pull/139)

Reply:
I thought it was a case of changing the blocks as requested in the spreadsheet and appending question/answer to them to the relevant questions and answers. I don't see a discussion on that particular PR that suggests otherwise?

Reply:
I have updated these now 👍